### PR TITLE
first pass of Github Pages documentation

### PR DESCRIPTION
**Pull request by nickgaski**

### CODE REVIEW

**Comment by nickgaski** [comment]
This will still need to be integrated with GitHub pages.  -> Settings -> Pages -> choose `/docs` as the root source.

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: Home 
+nav_order: 1
+---
+
+# Firefly
```
Comment (nguyer) [inline docs/index.md:7]:
```suggestion
# FireFly
```

**Review by nguyer — CHANGES_REQUESTED** [review]
Please make sure your commit includes proper sign-off. You can amend the existing commit by running:
```git commit --amend -s```

You may need to force push to your fork after that.

**Comment by nickgaski** [comment]
rebasing

**Comment by shorsher** [comment]
closing in favor of https://github.com/hyperledger-labs/firefly/pull/53

**Pull request closed without merging**